docs/tutorials/embed/mvmds_proof.ipynb

```diff
@@ -22,8 +22,8 @@
    "metadata": {},
```

Comment:
why is mvmds_proof being changed in this PR?

Reply:
Accidental haha. Going to change back